stockflux-search/public/app.dev.json

```diff
@@ -1,7 +1,7 @@
 {
     "runtime": {
         "arguments": "--enable-aggressive-domstorage-flushing",
-        "version": "9.61.38.40"
+        "version": "10.66.39.43"
```

Comment:
Think we can actually use "stable" in here, but that may be a risk if there're breaking changes. It'd be nice to think that this repo will be kept at the bleeding edge going forward though.